.gitignore

```diff
@@ -21,3 +21,4 @@ pcre_constants.include
 junk
 
 .vagrant/
+my_test.ngs
```

Comment:
Please make sure the lines that you add at the end of the file have EOL (\n) character in the end